bot/__main__.py

```diff
@@ -1,9 +1,12 @@
 import asyncio
+import logging
 
 from bot import config
 from bot.bot import WarnetBot
 from bot.config.logger import setup_logger
 
+logging.getLogger("hoyolabrssfeeds").setLevel(logging.CRITICAL + 1)
```

Comment:
I think we should keep an error log. At a minimum, we'll need error-level logging to identify any issues that may arise from the library itself.